src/index.ts

```diff
@@ -75,6 +80,9 @@ export class Authenticator {
     if ('httpOnly' in params && typeof params.httpOnly !== 'boolean') {
       throw new Error('Expected params.httpOnly to be a boolean');
     }
+    if ('sameSite' in params && !['Strict', 'Lax', 'None'].includes(params.sameSite)) {
```

Comment:
It might to be a good idea to define `SameSite` as a String enum instead of union. So that you don't have to list possible values twice.